tests/mock_crust/mod.rs

```diff
@@ -249,9 +251,11 @@ fn simultaneous_joining_nodes(
         // Set the specified relocation interval on the nodes of the given prefixes
         let relocation_interval = setup
             .dst_relocation_interval_prefix
-            .map(|prefix| (prefix.lower_bound(), prefix.upper_bound()));
-        nodes_with_prefix_mut(&mut nodes, &setup.dst_section_prefix)
-            .for_each(|node| node.inner.set_next_relocation_interval(relocation_interval));
+            .map(|prefix| XorTargetInterval(prefix.lower_bound(), prefix.upper_bound()));
+        nodes_with_prefix_mut(&mut nodes, &setup.dst_section_prefix).for_each(|node| {
+            node.inner
+                .set_next_relocation_interval(relocation_interval.clone())
```

Comment:
Can we `#[derive(Copy)]` for `XorTargetInterval`?

Reply:
We could, but would it be a good thing?

Reply:
It would avoid this `clone` here, for one thing. But not sure. `RangeInclusive` is not `Copy` so maybe this shouldn't be either? Up to you.

Reply:
I'll leave it out for now then.